Updated the new section on creating a service dialog (using the edito…

suyogsainkar opened the pull request:
…r) to match the UI options. Removed the older procedure of creating service dialog.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1477946

cbudz commented on common/service-dialog-creation-using-dialog-editor.adoc:
```diff
@@ -1,11 +1,11 @@
-{product-title} adds a new drag-and-drop service dialog editor. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog - dialog tabs, boxes, and elements - to help you easily use them to design your dialog.
+{product-title} adds a new drag-and-drop service dialog editor to create service dialogs. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog, that is dialog tabs, sections (previously referred as boxes), and elements to help you easily use them to design your dialog.
```
* I do this all the time - but I would refrain from referring to a new feature as 'new' in the documentation itself, since technically, by the next release it's no longer new and not something you want to have to double back to remove down the road. Definitely a good feature synopsis for the release notes, though. 

* The second sentence might be best parceled out into two. Maybe something like "The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog. Easily design your dialog utlizing dialog tabs, secions (previously....) and elements.

suyogsainkar replied:
Ack, fixed.

cbudz commented on common/service-dialog-creation-using-dialog-editor.adoc:
```diff
@@ -1,11 +1,11 @@
 
 * One or more *Tabs*.
-* Inside the tabs, one or more *Sections*. Note that in the previous version of {product-title_short}, *Sections* were referred as *Boxes*.
+* Inside the tabs, one or more *Sections*. Note that in the previous method of creating service dialogs using the {product-title_short} user interface, *Sections* were referred as *Boxes*.
```
"to" missing after referred.

suyogsainkar replied:
Fixed.

cbudz commented on common/service-dialog-creation-using-dialog-editor.adoc:
```diff
@@ -1,11 +1,11 @@
+{product-title} adds a new drag-and-drop service dialog editor to create service dialogs. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog, that is dialog tabs, sections (previously referred as boxes), and elements to help you easily use them to design your dialog.
 
-Using a service dialog, you can determine which options the users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic Red Hat Enterprise Linux 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or ...
+Using a service dialog, you can determine what options users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic Red Hat Enterprise Linux 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or othe ...
```
I would recommend some revision to this paragraph. Perhaps restructure to state that 'options and fields should be determined based on requirements for the service ordered. Then I'd utlize each of the examples you've submitted. In case #1 (RHEL) users may want to determine a, b, c. Then case #2, where options are more of a hard requirement as they impact the success of an ordered service. 
Let me know if you agree.

suyogsainkar replied:
Chris, thanks for the suggestion. I've updated the text a bit as well as referenced the example in the appendix (unable to xref the example at the moment as the other PR #578 is yet to be merged).

suyogsainkar commented:
@cbudz Hi Chris, thanks for your review and comments. I've addressed the feedback items in this PR, though my individual comments from last night seem to have disappeared. Would you mind taking a quick look and merging for the beta release? Let me know what you think. Thanks, Suyog

cbudz commented:
Hey Suyog,

Revisions look great. Merging this PR.
-Chris
